vault-tool/vault-tool.cabal

```diff
@@ -20,7 +20,9 @@ source-repository head
 
 library
   exposed-modules:     Network.VaultTool,
-                       Network.VaultTool.KeyValueV2
+                       Network.VaultTool.KeyValueV2,
+                       Network.VaultTool.TOTP,
+                       Data.Aeson.Utils
```

Comment:
IMO we should move `Data.Aeson.Utils` to `other-modules`.  It is not part of the package API, and hiding it means we can make arbitrary changes whenever we want.

Reply:
a1264907164eaf944749486033a3e9c1df1c6bf5